tests/testthat/test_AffineScoreFunction.R

```diff
@@ -76,20 +85,36 @@ test_that("Arithmetic for conditional scores works", {
     incpt  <- 1
     afcosc <- AffineConditionalScore(c(afcp, css), c(20, .1), 1.1)
 
+    expect_equal(
+        evaluate(afcosc, design, z1),
+        20 * evaluate(afcp, design, z1) + .1 * evaluate(css, design, z1) + 1.1
+    ) # arithmetic works
+
+    expect_equal(
+        evaluate(afcp + css, design, z1),
+        evaluate(css + afcp, design, z1)
+    ) # addition is commutative
+
     expect_equal(
         afcosc + incpt,
         incpt + afcosc
-    ) # addition is commutative
+    ) # addition with numeric is commutative
 
     expect_equal(
         afcosc * incpt,
         afcosc
-    ) # multiplication by 1 does not change
+    ) # multiplication by 1 does not change anything
 
     expect_equal(
         incpt * afcosc,
         afcosc
-    ) # multipilcation is commutative
+    ) # multiplication is commutative
+
+    expect_equal(
+        evaluate((-1) * afcp + afcosc, design, z1),
```

Comment:
do we need the brackets here? Does `-1 afcp` work? We should create an issue for getting -score to work at some point (non-priority)...

Reply:
#64